game/Assets/_Lvl2 Scripts/Lvl2GameController.cs

```diff
@@ -8,15 +8,18 @@ public class Lvl2GameController : MonoBehaviour {
 
     //public
     public AudioSource backgroundmusic;
+    public GameObject Player;
+    public Transform SpawnPoint;
 
     // Use this for initialization
     void Start () {
         _lights = GameObject.FindGameObjectsWithTag("Light");
         foreach (GameObject light in _lights)
         {
             var source = light.GetComponent<Light>();
```

Comment:
when it works it works
i have tried using Light or GameObject but none of the two work but with var it works